docs/community/3.15-announcement.md

```diff
@@ -0,0 +1,58 @@
+<style>
+.promo li a {
+    float: left;
+    width: 130px;
+    height: 20px;
+    text-align: center;
+    margin: 10px 30px;
+    padding: 150px 0 0 0;
+    background-position: 0 50%;
+    background-size: 130px auto;
+    background-repeat: no-repeat;
+    font-size: 120%;
+    color: black;
+}
+.promo li {
+    list-style: none;
+}
+</style>
+
+# Django REST framework 3.15
+
+At the Internet, on January 12th, 2024, with 135 commits 107 authors, we are happy to announce the release of Django REST framework 3.15.
+
+## Django 4.2 and Python 3.11 support
+
+The latest release now fully supports Django 4.2 and Python 3.11.
+
+The current minimum versions of Django still is 3.0 and Python 3.6.
```

Comment:
The current minimum versions of Django still is 3.0 and Python 3.7.

Reply:
from what I can see in the code, it's actually 3.6, e.g https://github.com/encode/django-rest-framework/blob/master/setup.py#L87

Reply:
That was, given that my 3.6 removal PR was taken into this release. But the PR now removes 3.6 and 3.7, and given DRF still officialy supports django 3.0. which supports python 3.6 and 3.7, we can't take it for the 3.15 release.

Reply:
I think Django 3.0 can be supported with Python 3.7 only, it doesn't seems a bad thing to me

Reply:
what's the point of doing that?

Reply:
It would alleviate the CI while still accepting bugs from Django 3.0.

Dropping support for a version means "no bugs on that stack will be addressed nor the backward compatibility will be checked" to me.

In the case of a bug arising from Django 3 and Python 3.6, it should be encouraged to upgrade Python and see if it persists.

It would still be compatible with your PR as Django 3 supports Python 3.6-3.9 (https://docs.djangoproject.com/en/5.0/releases/3.0/). With your PR, the minimum version of the stack checked (or fully supported) would be Django >= 3, Python >= 3.8

Reply:
> The current minimum versions of Django is 3.0 and Python 3.7.

What makes you say that? We've currently still got the test suite running against 3.6, am I missing something here?

Reply:
Moving to [Django 4.0+ and Python 3.8+](https://docs.djangoproject.com/en/5.0/releases/4.0/#python-compatibility) would prob be sensible.

3.2 LTS is almost at EOL... https://www.djangoproject.com/download/#supported-versions

We could then tie into their release cadence.

Reply:
I think keeping support of django 3.2 LTS would be a good idea now. we should keep two LTS working for easier upgrades from existing projects.